xml/art_virtualization-best-practices.xml

```diff
@@ -478,11 +478,11 @@ always madvise [never]</screen>
      KSM daemon <systemitem class="daemon">ksmd</systemitem> periodically scans
      user memory looking for pages of identical content which can be replaced
      by a single write-protected page. To enable the KSM service and start it,
-     run the following command:
+     run the following command
     </para>
 <screen>&prompt.user;&sudo; systemctl enable --now ksm.service</screen>
     <para>
-     Which will be available if the package <package>qemu-ksm</package>
+     which will be available if the package <package>qemu-ksm</package>
      is installed. Alternatively, it can also be started by running this other
```

Comment:
It would probably be best to
1. Not split the sentence around a screen at all.
2. Mention the required package before telling people to run a command from it.

GitHub is being weird with suggestions, so:

```
     by a single write-protected page.
     To enable the KSM service and start it, make sure the package
     <package>qemu-ksm</package> is installed. Then run the command:
    </para>
<screen>&prompt.user;&sudo; systemctl enable --now ksm.service</screen>
    <para>
     Alternatively, it can also be started by running this other
```

Reply:
Good points. TBH I thought this was _just_ a proofing pass and not one for editorial changes, but no time like the present. I will do this ASAP.

Reply:
Thanks!